### DISCUSSION TITLE
Add avg outside temperature on the speed rates dashboard

### DISCUSSION THREAD

**Discussion by aklira** [jheredianet/Teslamate-CustomGrafanaDashboards#99] [Ideas]
Hi,
Since lithium-ion batteries are sensitive to outside temperature, average consumption also depends on this factor.

It would be then really helpful to include the average outside temperature in the speed rates table on the speed rates dashboard.

**Comment by jheredianet** [comment]
There is a new dashboard "[Speed & Temperature](https://github.com/jheredianet/Teslamate-CustomGrafanaDashboards?tab=readme-ov-file#speed--temperature)" specially for this.
